### PR TITLE
Native inline videos

### PR DESCRIPTION
**Pull request by tuncaulubilge**
Inline video click callback added and inline video splash aspect ratio fixed for native

### CODE REVIEW

File: packages/article/src/article-body/article-body-row.js [hunk 1]
```diff
@@ -85,18 +84,22 @@ const ArticleRow = ({ content: { data, index }, onLinkPress }) =>
         caption
       }
     ) {
+      const aspectRatio = 16 / 9;
```
Comment (aronshamash) [inline packages/article/src/article-body/article-body-row.js]: I think we have this aspectRatio in various components. Would it be worth sharing it?

Comment (tuncaulubilge) [reply packages/article/src/article-body/article-body-row.js]: There is an AspectRatioContainer component with takes strings as props: https://github.com/newsuk/times-components/blob/master/packages/article/src/article-body/media-aspect-ratio.js#L6

This seems to be the only place where we use 16/9 as a hardcoded value, other than tests. We could have a shared set of constants for widely used aspect ratios throughout the project

Comment (aronshamash) [reply packages/article/src/article-body/article-body-row.js]: Also in related articles https://github.com/newsuk/times-components/blob/master/packages/related-articles/src/related-article-item.js#L39

Comment (tuncaulubilge) [reply packages/article/src/article-body/article-body-row.js]: https://github.com/newsuk/times-components/issues/895

Comment (aronshamash) [reply packages/article/src/article-body/article-body-row.js]: cool

**Pull request merged**